backend/test/test-setup.ts

```diff
@@ -149,10 +156,11 @@ export class TestSetupBuilder {
    */
   private static async createTestDatabase(dbName: string) {
     const dbType = process.env.HEDGEDOC_TEST_DB_TYPE || 'sqlite';
+    process.stdout.write(`Using dbname ${dbName} (${dbType})\n`);
```

Comment:
`console` is already hooked by the test framework or why do you use low-level writing? Maybe add a comment for the line.